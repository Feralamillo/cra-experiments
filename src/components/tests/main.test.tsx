import React from 'react';
import { shallow } from 'enzyme';

import { Main } from '../main';

jest.mock('../jumbotron', () => ({
  Jumbotron: 'mock-jumbotron',
}));
jest.mock('../card', () => ({
  Card: 'mock-card',
}));

test('renders the component', () => {
  const wrapper = shallow(<Main />);

  expect(wrapper).toMatchSnapshot();
});
