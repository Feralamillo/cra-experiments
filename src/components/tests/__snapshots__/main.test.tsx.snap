// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<main
  role="main"
>
  <mock-jumbotron />
  <div
    className="album py-5 bg-dark"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
        <mock-card />
      </div>
    </div>
  </div>
</main>
`;
